account_due_list/__manifest__.py

```diff
@@ -1,42 +1,32 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2011 Domsense srl (<http://www.domsense.com>)
-#    Copyright (C) 2011-2013 Agile Business Group sagl
-#    (<http://www.agilebg.com>)
-#    @author Jordi Esteve <jesteve@zikzakmedia.com>
-#    @author Lorenzo Battistini <lorenzo.battistini@agilebg.com>
-#    Ported to OpenERP 7.0 by Alex Comba <alex.comba@agilebg.com> and
-#    Bruno Bottacini <bruno.bottacini@dorella.com>
-#    Ported to Odoo by Andrea Cometa <info@andreacometa.it>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2008 Zikzakmedia S.L. (http://zikzakmedia.com)
+#        Jordi Esteve <jesteve@zikzakmedia.com>
+# © 2011 Domsense srl (<http://www.domsense.com>)
+# © 2011-2017 Agile Business Group (<http://www.agilebg.com>)
+# © 2015 Andrea Cometa <info@andreacometa.it>
+# © 2015 Eneko Lacunza <elacunza@binovo.es>
+# © 2015 Tecnativa (http://www.tecnativa.com)
+# © 2016 Eficent Business and IT Consulting Services S.L.
+#        (http://www.eficent.com)
+# © 2016 David Dufresne <david.dufresne@savoirfairelinux.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
 {
     'name': "Payments Due list",
-    'version': '8.0.0.1.0',
+    'version': '10.0.0.1.0',
```

Comment:
I would rather change it at merging time, in order to better handle versions with setuptools-odoo